metricbeat/tests/system/test_processors.py

```diff
@@ -39,8 +39,10 @@ def test_drop_fields(self):
         ]), evt.keys())
         network = evt["system"]["network"]
         print(list(network.keys()))
-        self.assertCountEqual(self.de_dot(["name", "out", "in"]), network.keys())
+        self.assertCountEqual(self.de_dot(
+            ["name", "out", "in"]), network.keys())
 
+    # @unittest.skipUnless(re.match("(?i)linux|darwin|freebsd", sys.platform), "os")
```

Comment:
Either delete the line or keep it, but please do not comment it out.

Reply:
Yah, that's not final, I'm just trying to figure out what's  going on on Windows.